terraform/app/variables.tf

```
@@ -160,6 +160,7 @@ variable "enable_splunk" {
 variable "app_version" {
   type        = string
   description = "The version identifier for the MAVIS application deployment"
+  default     = "NA"
```

```suggestion
  default     = "Unknown"
```

Just thinking about what users might see in this case.

---

I think `Not available` is at least better than `NA` in that case.

---

I agree with @thomasleese, if this is for user consumption then words are better than just a couple letters.